### DISCUSSION TITLE
Heading not showing up??

### DISCUSSION THREAD

**Discussion by sanders112** [iNavFlight/inav#8298] [Q&A] [answered]
I cannot get the heading to show up.  It won't tell me if I'm headed north, south, etc.  I only see dots  Does anyone have a suggestion?

**Comment by sanders112** [comment, marked as answer]
Uploaded the default font, even though I just flashed new firmware and it fixed it.

**Reply by MrD-RC** [reply]
Fonts are not changed with the firmware. You should upload the font when you flash a new major version.
